stripe-crypto-onramp/src/main/java/com/stripe/android/crypto/onramp/CheckoutState.kt

```diff
@@ -0,0 +1,36 @@
+package com.stripe.android.crypto.onramp
+
+import com.stripe.android.crypto.onramp.model.OnrampCheckoutResult
+import com.stripe.android.model.PaymentIntent
+
+/**
+ * Represents the checkout state with session info and current status.
+ */
+internal data class CheckoutState(
+    val onrampSessionId: String? = null,
```

Comment:
I think it should be:
 1. `onrampSessionId` should be non-null.
 2. `checkoutState: CheckoutState?` should be a nullable prop in `OnrampState` to mean there is no checkout in progress
 3. Remove `Idle` status.

```suggestion
internal data class CheckoutState(
    val onrampSessionId: String,
```